website/static/js/profile.js

```diff
@@ -550,13 +577,68 @@ var SocialViewModel = function(urls, modes) {
         return false;
     });
 
+    self.addWebsite = function() {
```

Comment:
This name could be improved; `addWebsite` implies that the method is used to add a new website to a user's profile; in reality, this adds a blank input. Maybe `addWebsiteInput`?